docs/product-user-guide/src/main/asciidoc/drl-rules-central-create-proc.adoc

```diff
@@ -0,0 +1,182 @@
+[id='drl-rules-central-create-proc']
+= Creating DRL files in {CENTRAL}
+
+You can create and manage DRL files for your project in {CENTRAL}. In each DRL file, you define rule conditions, actions, and other components related to the rule, based on the data objects you create or import in the package.
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *DRL file*.
+. Enter an informative *DRL file* name and select the appropriate *Package*. The package that you specify must be the same package where the required data objects have been assigned or will be assigned.
++
+You can also select *Use Domain Specific Language (DSL)* if any DSL assets have been defined in your project (in the *Domain Specific Language Definitions* panel in the *Project Explorer*). These DSL assets will then become usable objects for conditions and actions that you define in the DRL designer.
++
+. Click *Ok* to create the rule asset.
++
+The new DRL file is now listed in the *DRL* panel of the *Project Explorer*, or in the *DSLR* panel if you selected the *Use Domain Specific Language (DSL)* option. The package to which you assigned this DRL file is listed at the top of the file.
++
+. In the *Fact types* list in the left panel of the DRL designer, confirm that all data objects and data object fields (expand each) required for your rules are listed. If not, you can either import relevant data objects from other packages by using `import` statements in the DRL file, or xref:data-objects-create-proc_drl-rules[create new data objects] within your package. *<@Michael (1 of 13): This correct, for importing data objects from other packages? That's what it seems to me. See also the Import bullet below, as a result.>*
+. After all data objects are in place, return to the *Editor* tab of the DRL designer and define the DRL file with any of the following components:
++
+.Components of a DRL file
+[source]
+----
+package  //automatic
+
+import
+
+global  //optional
+
+function  //optional
+
+query  //optional
+
+declare   //optional
+
+rule
+
+rule
+
+...
+
+----
++
+*<@Michael (2 of 13): I was going to add to this list the keyword `template`, since it's in the list of supported DRL keywords in the Drools doc. I assumed it allowed you to add a rule template to a DRL file, with template keys and such, but I don't think that's right. Right? Again, template keys only apply to rule assets that have an accompanying data table where the values are defined, like guided decision tables and guided rule templates. So template keys are not supported in DRL files, nor `template` keyword, correct? If the `template` keyword is supported, can you tell me how it's used then?>*
```

Comment:
Gotcha. Since I've already documented template keys in other docs where the embedded DRL designer actually applies, I don't want to add it here. I'll consider mentioning it in the other creating sections (Dev Studio, Java, Maven), if it makes sense to do so. Will think on that. Thanks.